src/main/kotlin/day14/PracDatasource.kt

```
@@ -0,0 +1,1254 @@
+
+
+val json = """[{
+    "userId": 1,
+    "id": 1,
+    "title": "delectus aut autem",
+    "completed": false
+}]
+"""
+
+@Serializable
+data class Todo(val userId: Int?, val id: Int?, val title: String?, val completed: Boolean?) {}
```

detekt (1.23.7)

[warning] The class or object Todo is empty.

(detekt.empty-blocks.EmptyClassBlock)